test/js/ez-selection-activate-element.js

```diff
@@ -0,0 +1,117 @@
+        }));
+
+        assert.isFalse(preventDefaultCalled);
+    }
+
+    function checkAndclickEveryElements(elem, selector, isChecked) {
+        const checkboxes = elem.querySelectorAll(selector);
+        // same comment than in checkAndclickOneElement().
+        let preventDefaultCalled = false;
+        const prevent = function(e) {
+            preventDefaultCalled = e.defaultPrevented;
+            e.preventDefault();
+        };
+
+        document.addEventListener('click', prevent);
+        checkboxes.forEach((checkbox) => {
+            checkbox.checked = isChecked;
+            checkbox.dispatchEvent(new CustomEvent('click', {
+                bubbles: true,
+            }));
+        });
+
+        assert.isFalse(preventDefaultCalled);
+    }
+
+    it('should be defined', function () {
+        assert.equal(
+            window.customElements.get('ez-selection-activate-element'),
+            element.constructor
+        );
+    });
+
+    describe('properties', function () {
+        describe('`selectionSelector`', function () {
+            it('should default to `input[type="checkbox"]``', function () {
+                assert.equal(element.selectionSelector, 'input[type="checkbox"]');
+            });
+        });
+    });
+
+    describe('activate elements', function () {
+        describe('single selection', function () {
+            it('should enable activable buttons', function () {
+                const activableElement = element.querySelector('.ez-js-activable-element');
+
+                checkAndclickOneElement(element, element.selectionSelector, true);
+                assert.isFalse(activableElement.disabled);
+            });
+
+            it('should enable activable buttons based on custom selection', function () {
+                const activableElement = elementSubsetElementChecked.querySelector('.ez-js-activable-element');
+
+                checkAndclickOneElement(elementSubsetElementChecked, elementSubsetElementChecked.selectionSelector, true);
+                assert.isFalse(activableElement.disabled);
+            });
+
+            it('should not enable if the activable button does not match the custom selection', function () {
+                const activableElement = elementSubsetElementChecked.querySelector('.ez-js-activable-element');
+
+                checkAndclickEveryElements(elementSubsetElementChecked, '.this-checkbox-will-not-match', true);
+                assert.isTrue(activableElement.disabled);
+            });
+        });
+
+        describe('multiple selection', function () {
+            it('should enable activable button', function () {
+                const activableElement = element.querySelector('.ez-js-activable-element');
+
+                checkAndclickEveryElements(element, element.selectionSelector, true);
+                assert.isFalse(activableElement.disabled);
+            });
+
+            it('should disable button bound to a single selection', function () {
```

Comment:
to be consistent with the 'single selection' test outline, I think you should group those 2 tests.
And you should also add a `describe('custom selection')` to test custom selection and multiple single as well (which means in `TestFixtureSubsetElementChecked` fixture you'll need 2 checkboxes with the class `this-checkbox-will-not-match`)